src/common/POFields/FieldVendor.js

```diff
@@ -3,18 +3,27 @@ import PropTypes from 'prop-types';
 
 import FieldSelection from '../FieldSelection';
 
-const FieldVendor = ({ vendors }) => {
+const FieldVendor = ({ vendors, name, required, validate }) => {
   return (
     <FieldSelection
       dataOptions={vendors}
       labelId="ui-orders.orderDetails.vendor"
-      name="vendorName"
+      name={name}
+      required={required}
```

Comment:
I'm not sure we should validate vendor on templates

Reply:
When need create/edit order you must have vendor

Reply:
let's keep it as required for now